jsons/serializers/default_iterable.py

```diff
@@ -1,16 +1,72 @@
 from collections.abc import Iterable
+from multiprocessing import Process
+from typing import Tuple
+
+from typish import get_args
+
 from jsons._dump_impl import dump
+from jsons._multitasking import multi_task
+from jsons.exceptions import SerializationError
 
 
-def default_iterable_serializer(obj: Iterable, **kwargs) -> list:
+def default_iterable_serializer(
+        obj: Iterable,
+        cls: type = None,
+        *,
+        strict: bool = False,
```

Comment:
I don't know what else it affects but I would prefer to have `strict = True` by default.